src_c/event.c

```diff
@@ -590,6 +620,13 @@ pgEvent_AutoInit(PyObject *self, PyObject *_null)
     if (!_pg_event_is_init) {
```

Comment:
What about `pg_EnableKeyRepeat` and `pg_GetKeyRepeat`? Can they be called from another thread than the one that is currently running code from `pg_event_filter`?

Reply:
Yes, those can be called from a different thread (the main thread).. They modify state that the event filter reads (but does not write)... So I thought a mutex is not needed in this case... but maybe for extra safety it can be added I agree